src/views/Error.js

```diff
@@ -36,26 +36,44 @@ const CriticalError = () => {
             : translate('CriticalError.bodyCriticalErrorTryAgain')}
         </Typography>
         <AppNote messageVersion="error" />
-        {app.errorMessage && (
+        {!app.errorMessage && (
```

Comment:
Is this a mistake? Does make sense here to check the condition `!app.errorMessage`?

Reply:
thx for noticing it